src/Mutators/.coverage

```diff
@@ -0,0 +1 @@
+!coverage.py: This is a private format, don't read it directly!{"lines":{"/home/chai/crashsim/lib/python2.7/site-packages/sysDef/__init__.py":[1],"/home/chai/crashsim/lib/python2.7/site-packages/posix_omni_parser/parsers/__init__.py":[1],"/home/chai/crashsim/lib/python2.7/site-packages/posix_omni_parser/Syscall.py":[172,11,13,142,15,145,146,148,149,175,30,159,32,161,162,37,166,168,169,170,44,173,47,176,178,179,52,182,190,160,109,95,100,103,18,112],"/home/chai/crashsim/lib/python2.7/site-packages/posix_omni_parser/__init__.py":[1],"/home/chai/rrapper/src/mutator/UnusualFiletype.py":[19,1,2,3,5,6,7,40,9,10,11,34,14,15,35,20,22,25,26,38,28],"/home/chai/crashsim/lib/python2.7/site-packages/posix_omni_parser/Trace.py":[84,96,98,103,106,23,109,112,18,20,21,118,55,88,57,26,92,85],"/home/chai/rrapper/src/mutator/MutationError.py":[1,2],"/home/chai/crashsim/lib/python2.7/site-packages/posix_omni_parser/parsers/Parser.py":[12,14,17,19,40,44,48,57,61,64,67,71,98,101,102,105,106,107,108,111,113,116,120,123,124,125],"/home/chai/crashsim/lib/python2.7/site-packages/posix_omni_parser/parsing_classes.py":[15,17,19,21,22,25,31,32,33,549,555,558,47,560,562,564,574,64,68,72,77,81,87,88,603,607,611,615,105,106,621,623,625,116,117,617,635,639,642,644,133,134,648,649,650,651,652,654,656,657,658,659,154,155,669,672,673,681,684,687,689,217,218,219,222,223,232,235,236,250,255,256,272,277,278,48,298,301,303,330,335,338,340,351,354,356,367,370,372,383,384,472,500],"/home/chai/crashsim/lib/python2.7/site-packages/sysDef/SyscallParameter.py":[1,50,52,220,169],"/home/chai/rrapper/src/mutator/mutator.py":[1,3,4,38,41,45,77,81,51,21,22,68,26,53],"/home/chai/crashsim/lib/python2.7/site-packages/sysDef/SyscallManual.py":[65,39,41,42,43,108,45,110,109,141,49,114,51,436,105,408,111,61,62,426],"/home/chai/crashsim/lib/python2.7/site-packages/sysDef/Definition.py":[24,1,3,22,95],"/home/chai/crashsim/lib/python2.7/site-packages/posix_omni_parser/parsers/StraceParser.py":[1028,1031,523,528,88,535,536,27,29,542,31,32,35,38,563,568,570,573,574,693,577,67,581,584,588,592,593,594,596,598,600,103,104,105,621,119,122,151,152,154,159,675,677,680,681,687,178,692,181,698,699,722,742,894,811,776,778,779,783,784,785,786,789,792,795,810,903,814,815,822,823,315,316,317,318,319,320,321,322,837,539,328,329,331,336,337,852,341,346,859,862,351,865,354,355,361,366,371,372,887,891,380,381,382,383,384,897,901,391,906,398,399,404,413,416,419,65,423,937,428,429,431,952,968,971,163,468,992,848,422,1008],"/home/chai/rrapper/src/mutator/test_unusual.py":[1,2,3,4,6,7,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,26]}}
```

Comment:
This looks like a config file for your editor or something.  Please remove it from this commit